aspnet/mobile/native-mobile-backend.rst

```diff
@@ -1,11 +1,170 @@
-.. include:: /../common/stub-topic.txt
+:version: 1.0.0
 
-.. _native-mobile-backend:
+Creating Backend Services for Native Mobile Applications
+========================================================
 
-|stub-icon| Creating Backend Services for Native Mobile Applications
-====================================================================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+Mobile apps can easily communicate with ASP.NET Core backend services.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/mobile/native-mobile-backend/sample>`__
+
+The Sample App
```

Comment:
Multiple meanings of "sample app/code" can be confusing.  Above. there's a link to "sample code" and here there's a heading about "the sample app"; readers would assume this section explains what the "sample code" link above goes to.  Might be worth clarifying what each "sample" is and perhaps use more specific terminology for them.  E.g., don't say just "sample app" or "sample code" but rather something like sample ASP.NET Core service, sample project with native clients and a web api 2 service. 

It might also help to give an overview of what setup is going to be required.  The link "Xamarin Forms TodoRest app" doesn't go to an app; it goes to an article about an app with links to download a project that you have to open and run and install Xamarin tooling for in VS.